### PR TITLE
Automate notebook tests with gitlab.

### PR DESCRIPTION
**Pull request by DailyDreaming**
This adds a gitlab yaml.  All hooks and credentials should be set up.

Depends on #29 .

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+image: ubuntu:18.04
```
Comment (xbrianh) [inline .gitlab-ci.yml:1]: Can "image" be set directly to `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.12`?

**Review by xbrianh — APPROVED** [review]
I like the approach of creating a separate CI/CD job per notebook. However, instead of creating a `.gitlab-ci.yml` entry for each notebook, it would be preferable to define a general rule, if possible. 

This can be handled with a new issue and future PR.

File: .gitlab-ci.yml (hunk 1 above)
Comment (DailyDreaming) [reply .gitlab-ci.yml:1]: It can.  Will change.

**Review by xbrianh — APPROVED** [review]
lgtm

Consider making an issue to investigate how to pattern match in `.gitlab-ci.yml` to avoid manual entries for each notebook.

**Pull request merged**